### DISCUSSION TITLE
Soft wrap

### DISCUSSION THREAD

**Discussion by Underday** [community/community#4748] [Mobile] [answered]
I wonder why there isn't any option to use soft wrap?

**Comment by brianlovin** [comment, marked as answer]
This should be possible if you go to `Profile → Settings → Code Options` – there's a toggle to enable/disable line wrapping. Let me know if this works for you 😊

**Reply by Underday** [reply]
I know this feature, but it is hard wrap, I want soft wrap.

I don't want words to be cut in half

**Reply by brianlovin** [reply]
Got it, thanks for the clarification. I'll make an issue for this to track.

**Reply by Underday** [reply]
> Got it, thanks for the clarification. I'll make an issue for this to track.

It's been a long time but any progress?

**Reply by lizzydavis695** [reply]
Hi I need a little help
